eng/scripts/docs/Docs-ToC.ps1

```diff
@@ -21,3 +21,189 @@ function Get-NamepspacesFromDll($dllPath) {
 }
+
+function DownloadNugetPackage($package, $version, $destination) {
+    # $PackageSourceOverride is a global variable provided in
+    # Update-DocsMsToc.ps1. Its value can set a "customSource" property.
+    # If it is empty then the property is not overridden.
+    $customPackageSource = Get-Variable -Name 'PackageSourceOverride' -ValueOnly -ErrorAction 'Ignore'
+    # Download package from nuget, if no package found in nuget, then search devops public feeds if set.
+    LogDebug "Downloading source: '$customPackageSource'. If it is empty, we default to use nuget gallery."
+    if ($customPackageSource) {
+        nuget install -FallbackSource $customPackageSource $package -Version $version -DirectDownload -DependencyVersion Ignore -OutputDirectory $destination
+    }
+    else {
+        nuget install $package -Version $version -DirectDownload -DependencyVersion Ignore -OutputDirectory $destination
+    }
+}
+
+function Fetch-NamespacesFromNupkg ($package, $version) {
+    $tempLocation = (Join-Path ([System.IO.Path]::GetTempPath()) "extractNupkg")
+    if (!(Test-Path $tempLocation)) {
+        New-Item -ItemType Directory -Path $tempLocation -Force | Out-Null
+    }
+    LogDebug "Downloading nupkg package $package with version $version to $tempLocation ...."
+    DownloadNugetPackage -package $package -version $version -destination $tempLocation | Out-Null
+    $packageFolder = "$package.$version"
+    if ($LASTEXITCODE -ne 0 -or !$(Test-Path $tempLocation/$packageFolder)) {
+        LogDebug "Did not download the package correctly. Skipping..."
+        return @()
+    }
+    LogDebug "Searching for dll file..."
+    $dllFiles = @()
+    $dllFileName = ""
+    if (Test-Path "$tempLocation/$packageFolder/lib/netstandard2.0/"){
+        $dllFiles = @(Get-ChildItem "$tempLocation/$packageFolder/lib/netstandard2.0/*" -Filter '*.dll' -Recurse)
+    }
+    if (!$dllFiles) {
+        LogWarning "Can't find any dll file from $tempLocation/$packageFolder with target netstandard2.0."
+        $dllFiles = Get-ChildItem "$tempLocation/$packageFolder/lib/*" -Filter '*.dll' -Recurse
+        if (!$dllFiles) {
+            LogError "Can't find any dll file from $tempLocation/$packageFolder."
+            return @()
+        }
+    }
+    elseif ($dllFiles.Count -gt 1) {
+        LogWarning "There are multiple dll files in target netstandard2.0. "
+        if (Test-Path "$tempLocation/$packageFolder/lib/netstandard2.0/$package.dll") {
+            LogDebug "Use the dll file $package.dll"
+            $dllFileName = "$tempLocation/$packageFolder/lib/netstandard2.0/$package.dll"
+        }
+    }
+    if (!$dllFileName) {
+        $dllFiles = $dllFiles | Get-Unique | Sort-Object
+        $dllFileName = $dllFiles[0].FullName
+    }
+    LogDebug "Dll file found: $dllFileName"
+    $namespaces = Get-NamespacesFromDll $dllFileName
+    if (!$namespaces) {
+        LogError "Can't find namespaces from dll file $dllFileName." -ErrorAction Continue
```

Comment:
Looks like this ErrorAction is leftover and should be removed. Looks like there might be a few of them left over.